### DISCUSSION TITLE
Invalidate results based on absence of enhance_using_context results

### DISCUSSION THREAD

**Discussion by storkeu** [microsoft/presidio#1155] [Q&A]
Hi,
I have a case where two pattern recognizers will detect that a given string matches the patern.
Both of them use checksum validation and both return true, so both return 1.0 score (pattern captures 9 numbers, last being check digit, but they can refer to different entities)
The difference is in context words
I would like to descrease confidence based on the results of enhance_using_context results (absence of...)
I dont now how, implementing `invalidate_result(self, pattern_text: str)` in my custom recognizer I could access the results of the context enhancer...
I would like to have some clues or ideias. Is this possible? Makes sense?
TY

**Comment by omri374** [comment]
Hi, you could consider changing the `EntityRecognizer.MAX_SCORE` parameter to less than 1.0. In that case, the validation would not result in a confidence of 1.0, and context words would be able to differentiate between the two results. 
Another option is to post process the results based on the decision process (if you call the `analyze` method with `return_decision_process=True`, which includes the found context words).

**Comment by storkeu** [comment]
Hi, tks.
About the 1st option. I don't see where or how to change `EntityRecognizer.MAX_SCORE` It's hard set to `1.0` in `EntityRecognizer` I think.

**Comment by omri374** [comment]
Hi, you can change it globally:

```python
from presidio_analyzer import AnalyzerEngine, EntityRecognizer

text = "AL47212110090000000235698741" # valid IBAN number

analyzer = AnalyzerEngine()
EntityRecognizer.MAX_SCORE = 0.795

analyzer.analyze(text=text, language="en")

```
Results:
`[type: IBAN_CODE, start: 0, end: 28, score: 0.795]`

Note that this would change the validation score globally for all recognizers.
